editEventTitle.php

```diff
@@ -1,42 +1,46 @@
 <?php
 
 require_once('bdd.php');
+
 if (isset($_POST['delete']) && isset($_POST['id'])){
 	
+	$id = "";
 	
-	$id = $_POST['id'];
+	$id = test_input($_POST['id']);
 	
 	$sql = "DELETE FROM events WHERE id = $id";
 	$query = $bdd->prepare( $sql );
 	if ($query == false) {
 	 print_r($bdd->errorInfo());
-	 die ('Erreur prepare');
+	 die ('Error prepare');
 	}
 	$res = $query->execute();
 	if ($res == false) {
 	 print_r($query->errorInfo());
-	 die ('Erreur execute');
+	 die ('Error execute');
 	}
 	
 }elseif (isset($_POST['title']) && isset($_POST['description']) && isset($_POST['color']) && isset($_POST['id'])){
 	
-	$id = $_POST['id'];
-	$title = $_POST['title'];
-	$description = $_POST['description'];
-	$color = $_POST['color'];
+	$id = $title = $description = $color = "";
+	
+	$id = test_input($_POST['id']);

```

Comment:
please additionally implement for editEventTitle-json, test, and add to your branch